server/test/ping.spec.js

```diff
@@ -1,21 +1,29 @@
+require("dotenv").config();
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const app = require("../app.js");
+const express = require("express");
+
+const pingRouter = require("../routes/ping");
 
 chai.should();
 chai.use(chaiHttp);
 
-describe("/POST ping", function () {
-  it("it should return 400", function (done) {
+const { json } = express;
+const app = express();
+app.use(json());
+app.use("/ping", pingRouter);
+
+describe("/POST ping", () => {
+  it("it should return 400", (done) => {
     chai
       .request(app)
       .post(`/ping/`)
-      .send({ teamName: "Shums" })
+      .send({ teamName: "Pikachu" })
       .end((err, res) => {
         res.should.have.status(400);
         res.body.should.have
           .property("response")
-          .eql("Shums is not part of the team. Modify your .env");
+          .eql("Pikachu is not part of the team. Modify your .env");
```

Comment:
I don't think we need the ping route anymore, unless you want to turn it more into a health check